$NetBSD: patch-ah,v 1.1 2005/03/14 14:48:20 rillig Exp $

g++-2.95.3 does not allow class members that have the same name as the
class.

--- src/highscor/highscor.cc.orig	Sun Jun  6 21:58:28 1999
+++ src/highscor/highscor.cc	Sat Mar 12 11:20:50 2005
@@ -75,7 +75,7 @@ HighScore::HighScore() {
 
 		if(highscore[i].Level)
 	        	sprintf(buffer, " %-10s%6d   %1d:%1d", highscore[i].Name,
-	        		highscore[i].Score, ((highscore[i].Level-1) / 3) + 1, ((highscore[i].Level-1) % 3) + 1);
+	        		highscore[i].S_Score, ((highscore[i].Level-1) / 3) + 1, ((highscore[i].Level-1) % 3) + 1);
 		else
 	        	sprintf(buffer, " %-10s%6d   0:0", "Empty", 0);
 
@@ -166,13 +166,13 @@ HighScore::HighScore(int score, int leve
 
         for(int i = 0; i < NUM_HIGHSCORES; i++) {
 
-        	if(score > highscore[i].Score) {
+        	if(score > highscore[i].S_Score) {
 
                 	memmove(&highscore[i + 1], &highscore[i],
                 		sizeof(Score) * (NUM_HIGHSCORES - i - 1));
 
                         strcpy(highscore[i].Name, GetName());
-                        highscore[i].Score = score;
+                        highscore[i].S_Score = score;
                         highscore[i].Level = level;
                         break;
         	}
